### PR TITLE
HIP draft - Helium Founders NFT Reward System

### PR DESCRIPTION
**Pull request by Criptosaurios**
Ed – rendered view of latest: https://github.com/helium/HIP/blob/e23cbae6f85affcc56aa0718d8094ed8cf7e8123/0000-template.md

Ed: Old draft – https://github.com/helium/HIP/blob/e23cbae6f85affcc56aa0718d8094ed8cf7e8123/0000-template.md

### CODE REVIEW

**Comment by Dangerousfish** [comment]
I like this. Possibility to expand into all active hotspots from a specific point in time?

Richest - Is this wallets with the highest earnings since ??? period, or is this a snapshot at a specific time of total HNT in wallets?

**Comment by Criptosaurios** [comment]
Thanks for your interest [Dangerousfish](https://github.com/Dangerousfish)
Any recommendation or suggestion is welcome. 
> I like this. Possibility to expand into all active hotspots from a specific point in time?
The idea is to take a snapshot of all active hotspot in same date when LORA devices will start to get new SubDao token $IOT. 
Of course, is necessary that modifications related to HIP55 be 100% completed at the Snapshot date.

> Richest - Is this wallets with the highest earnings since ??? period, or is this a snapshot at a specific time of total HNT in wallets?
This is proposed as a Snapshot of balance in wallets at specific Date/Time.

**Comment by jamiew** [comment]
Hi @Criptosaurios, thanks for your submission

I think this could use some more work before it is ready to merge:

- what exactly are you proposing here? Giving out NFTs to network participants? I've read through a few times and I'm not exactly clear
- if it is an NFT, how would that be issued to the recipients? The Helium blockchain does not support smart contracts or NFTs or similar

Lastly, this may not actually need to be a HIP if no changes are required to the Helium blockchain or core tokenomics

**Comment by Criptosaurios** [comment]
@jamiew thanks for your review and clarification request.
This proposal is supposed to be implemented after Helium get migrated to the new blockchain that will support Smart Contracts, as necessary by HIP51 DAO.
Then, NFTs will work as _digital certificates_ to get the rewards as proposed in this HIP DRAFT (and of course only those who meet the requirements proposed will get rewards recurrently). Allowing to trade it, and move from a wallet to other as necessary.

**Comment by Criptosaurios** [comment]
@jamiew 
--Lastly, this may not actually need to be a HIP if no changes are required to the Helium blockchain or core tokenomics.

This paragraph included in the HIP draft implies a modification to the Tokemonics:

"This proposal establishes that each subDAO that is incorporated into HELIUM after the approval of
HIP51, must contribute 15% of the issuance of its tokens (DNT) to assign rewards to NFTs in the
following way:
5% will be distributed among the NFTs assigned to Hotspots. 5% Will be distributed among the
NFTs assigned to Validators. 5% will be distributed among the NFTs assigned to the 20 richest
wallets in HNT"

**Comment by Criptosaurios** [comment]
@jamiew Hi! do you have any additional questions about my HIP proposal?

**Comment by edakturk14** [comment]
hey @Criptosaurios! as per our converstation on discord looking forward for the edits following the chats over there and I have allocated a 15 min spot for the HIP draft on the next community call thats on 31st of Aug.

**Comment by Criptosaurios** [comment]
> hey @Criptosaurios! as per our converstation on discord looking forward for the edits following the chats over there and I have allocated a 15 min spot for the HIP draft on the next community call thats on 31st of Aug.

@edakturk14 Thanks for the follow up!. Yes, I will update the draft the comming week.

**Comment by vincenzospaghetti** [comment]
HI @Criptosaurios did you get to present at the Community Call on the 31st? Can you summarize what happened here and follow with any updates to your HIP?

**Comment by Criptosaurios** [comment]
> HI @Criptosaurios did you get to present at the Community Call on the 31st? Can you summarize what happened here and follow with any updates to your HIP?

Hi @vincenzospaghetti ! Thanks for the follow up. Yes, we ( @RDCH106 and me) presented the Draft during the last Community Call. In fact was a great participation due no questions, suggestions or comments were made!!. The moderator said that our DRAFT will be numbered and channel created, unfortunately we already have 2 weeks waiting for this. This HIP is is a proposal from the community for the community, then we expect to modify the content during a discussion, prior to ask for vote.
Also, as you can see, we immediately answered the questions made by last HIP editor, more than 2 months ago. Finally, we asked on July/18 if they have more questions, but no answer received.

**Comment by RDCH106** [comment]
> > HI @Criptosaurios did you get to present at the Community Call on the 31st? Can you summarize what happened here and follow with any updates to your HIP?
> 
> Hi @vincenzospaghetti ! Thanks for the follow up. Yes, we ( @RDCH106 and me) presented the Draft during the last Community Call. In fact was a great participation due no questions, suggestions or comments were made!!. The moderator said that our DRAFT will be numbered and channel created, unfortunately we already have 2 weeks waiting for this. This HIP is is a proposal from the community for the community, then we expect to modify the content during a discussion, prior to ask for vote. Also, as you can see, we immediately answered the questions made by last HIP editor, more than 2 months ago. Finally, we asked on July/18 if they have more questions, but no answer received.

Correct.

@vincenzospaghetti you can find the HIP presentation in the commonity call here:

https://vimeo.com/745169851

The HIP wasn't change since August 24, and the community call was August 31. Nothing was changed since then and we keep waiting for the HIP number and the space where we can discuss it.

Thanks in advance and regards.

**Comment by Criptosaurios** [comment]
@vincenzospaghetti Did you read the Draft Update made by @RDCH106 previous to the Community call? He is an authorized collaborator of this draft.
In the update exist no mention of the word NFT (after months of feedback and discussion in the HIP-GENERAL channel). 
Also, the draft clearly explain the complementary rewards proposed to Active hotsopts and some richest wallets. This reward is proposed to be the 15% of the DNTs issued (Only mobile by now, but also from others subDAO comming to helium excluding IOT subdao obviusly because this will be the benefiaciary). This will have an impact on the tokemonics of each subDAO.

**Comment by vincenzospaghetti** [comment]
Hi @Criptosaurios - Thanks for clarifying the NFT component of the HIP. There are still outstanding questions and clarity for this HIP before it goes to the community. 

You need to do several edits to specifically the Summary, Motivation, and Unanswered Questions before this HIP is viable for the community to discuss:

- [ ] Edit the summary. A summary cannot be 'ask for discussion'. **A summary needs to tell the reader what the HIP is changing and proposing and why.** 
- [ ] The unanswered questions need to be revised. None of your proposal answers: Is there a better way to stop the trend of equipment being shut down? Your HIP is just offering the distribution of rewards. If you are making an argument for stopping the shut down of equipment this should look much different. 
- [ ] Your mention of oracles and validators is confusing. Per the passing of HIP 70, there will be one oracle deployment by Nova Labs, and there will be no validators. This HIP does not understand the nuances of the network. Please address this.
- [ ] You need to answer: What is your motivation here? Do you want higher rewards? Do you want a more efficient denylist? It is simply not clear what your motivation is nor why this was even proposed.

**Comment by Criptosaurios** [comment]
@vincenzospaghetti  I´m happy that these comments are public.
Are you asking me to respond to the unanswered questions? Really? Did you know that the own HIP for HIP has a lot of very critical unanswered questions? https://github.com/helium/HIP/blob/main/0007-managing-hip-process.md
That is the purpose of having a discussion of HIP.
Really you still don´t understand the motivation of this Draft? Is clear in the document and also what I said during the community call.
Question: Officially are you stopping our draft HIP to become a numbered HIP as is now?
Because looks like personal opinions. Our Draft meets all the requirements established here:
https://github.com/helium/HIP/blob/main/0007-managing-hip-process.md
@RDCH106

**Comment by RDCH106** [comment]
@vincenzospaghetti the whole procedure has been a waste of time and is an excuse to say that it is an open project when it really does not facilitate or help the introduction of contributions that do not come from within Helium.

The experience has not been good and the management has been very questionable. With this type of deal, it is to be expected that people end up not supporting the project. The price of the HNT and the number of online hotspots is an indisputable facts:

- https://explorer.helium.com/market
- https://explorer.helium.com/iot

The IOT network is close to dropping below 50% active hotspots. Our proposal was going first to open a debate and second try to propose a solution:

https://github.com/Cryptosaurs/DRAFT-HIP---LORAWANS-ACTIVE-MAINTENANCE-STIMULUS/blob/b4604b394562f33224a8234cf6e805beb9c46b5d/0070-lorawans-active-maintenance-stimulus.md

Both things have not happened.

@Criptosaurios
